notebooks/external_innervation.ipynb

```diff
@@ -300,6 +394,38 @@
     "ax.set_xticks([]); ax.set_yticks([])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Excercise\n",
+    "Calculate how many SC synapses a pyramidal cell or an interneuron receive on average. Plot the distribution and provide mean and std. Put a tuple of mean and std in 'ans_1a' (pyramidal cell) and 'ans_1b' (interneuron) respectively.\n",
+    "\n",
+    "The pandas dataframe syns contains only gid of the postsynaptic cells but not the mtype. For this information, you should need to load the circuit and create a Circuit object (see 2.5)"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
I would insert a cell just before that and indicate that this is a cell where the user can work the exercise.